base/multimedia.jl

```diff
@@ -93,6 +93,12 @@ for mime in ["text/vnd.graphviz", "text/latex", "text/calendar", "text/n3", "tex
     @eval @textmime $mime
 end
 
+immutable MIMEData{mime}
+    data::Vector{UInt8}
```

Comment:
make this untyped

Reply:
@vtjnash Why would you make the data untyped?  To work with `HTTP.jl` as a `Resource` you have to emit `Vector{UInt8}`.  The printing logic for `show` could use `istextmime` to determine if it should be shown as octets or as a string?

Reply:
the `write` method handles any type, and other users might want to use other array types, or String, etc. here

Reply:
Ok.  So, `write` fallback would have an if/then list based upon the type of the `data` attribute?

Reply:
Multiple dispatch already fully addresses that